env-defaults

```diff
@@ -104,6 +104,9 @@ MAX_SYNCHRONOUS_GRANULES=1
 # or maximum_async_granules we will override the configuration to limit to this value.
 MAX_GRANULE_LIMIT=2100
 
+# The threshold of the number of granules in a request that will trigger auto-pausing with preview
+PREVIEW_THRESHOLD=500
```

Comment:
The actual number was not specified in the AC. I can change this to whatever we think the default should be.